spec/domain/accounts/users_spec.exs

```diff
@@ -75,17 +75,17 @@ defmodule Squadster.Domain.Accounts.UsersSpec do
       it "returns map with data" do
         data = User.data_from_auth(auth())
 
-        expect(data.first_name).to_not(be_nil())
-        expect(data.last_name).to_not(be_nil())
-        expect(data.birth_date).to_not(be_nil())
-        expect(data.mobile_phone).to_not(be_nil())
-        expect(data.university).to_not(be_nil())
-        expect(data.faculty).to_not(be_nil())
-        expect(data.small_image_url).to_not(be_nil())
-        expect(data.image_url).to_not(be_nil())
-        expect(data.uid).to_not(be_nil())
-        expect(data.vk_url).to_not(be_nil())
-        expect(data.auth_token).to_not(be_nil())
+        expect(data.first_name) |> to_not(be_nil())
+        expect(data.last_name) |> to_not(be_nil())
+        expect(data.birth_date) |> to_not(be_nil())
+        expect(data.mobile_phone) |> to_not(be_nil())
+        expect(data.university) |> to_not(be_nil())
+        expect(data.faculty) |> to_not(be_nil())
+        expect(data.small_image_url) |> to_not(be_nil())
+        expect(data.image_url) |> to_not(be_nil())
+        expect(data.uid) |> to_not(be_nil())
+        expect(data.vk_url) |> to_not(be_nil())
+        expect(data.auth_token) |> to_not(be_nil())
```

Comment:
Nice!

Reply:
Do you have any idea why travis didn't fail?

Reply:
It failed locally and did not failed on travis?

Reply:
It depends on OTP version if I'm not mistaken, with 21+ Ruby syntax will fail, so maybe you have 21+ locally and Travis runs 20.3